### PR TITLE
kbfsgit: use handle type/name to check if tlf is managed by app

### PR DESCRIPTION
**Pull request by strib**
Previously we were using the number of users in the handle, which breaks for implicit teams.  So instead use the type, and for private TLFs, use the full name and see if multiple users are listed.

### CODE REVIEW

File: kbfsgit/runner.go [hunk 1]
```diff
@@ -253,6 +253,25 @@ func (r *runner) printDoneOrErr(
...
+func (r *runner) isManagedByApp() bool {
+	switch r.h.Type() {
+	case tlf.Public:
+		// Public TLFs are never managed by the app.
+		return false
+	case tlf.SingleTeam:
+		// Single-team TLFs are always managed by the app.
+		return true
+	case tlf.Private:
+		// Only single-user private TLFs are managed by the app.  So
```
Comment (jzila) [inline kbfsgit/runner.go]: What about single-user private TLFs that haven't yet been transitioned to implicit teams?

Comment (strib) [reply kbfsgit/runner.go]: What's the concern?  This function will correctly return `true` in that case as well, right?

Comment (strib) [reply kbfsgit/runner.go]: Yes, single-user private repos are managed by the app/CLI (i.e., the "personal" section of the GUI), along with team repos.

Comment (jzila) [reply kbfsgit/runner.go]: Ah, I see now what is meant by "managed". Thanks!

**Pull request merged**